### PR TITLE
Throw DdlException when use admin set frontend config

### PR DESCRIPTION
**Pull request by WingsGo**
When execute Ddl `admin set frontend config`. User will get msg `ERROR 1064 (HY000): Unexpected exception: null`, it is confusing, we should give user a more friendly tip info.

### CODE REVIEW

**Comment by WingsGo** [comment]
For #3538

File: fe/src/main/java/org/apache/doris/catalog/Catalog.java [hunk 1]
```diff
@@ -6385,7 +6385,10 @@ public void replayDropFunction(FunctionSearchDesc functionSearchDesc) {
 
     public void setConfig(AdminSetConfigStmt stmt) throws DdlException {
         Map<String, String> configs = stmt.getConfigs();
-        Preconditions.checkState(configs.size() == 1);
+        // configs can not be null
+        if (configs.size() != 1) {
```
Comment (morningman) [inline fe/src/main/java/org/apache/doris/catalog/Catalog.java:6389]: This should be done in `AdminSetConfigStmt`'s `analyze` method.

Comment (WingsGo) [reply fe/src/main/java/org/apache/doris/catalog/Catalog.java]: done

**Review by kangkaisen — DISMISSED** [review]
+1， LGTM

**Review by morningman — DISMISSED** [review]
LGTM

**Review by yangzhg — CHANGES_REQUESTED** [review]
`fe/src/test/java/org/apache/doris/analysis/AminSetConfigStmtTest` no file extension;

**Comment by WingsGo** [comment]
> `fe/src/test/java/org/apache/doris/analysis/AminSetConfigStmtTest` no file extension;

done

**Review by imay — APPROVED** [review]

**Pull request merged**